controller/controller.go

```diff
@@ -275,14 +280,16 @@ func (c *Controller) ProcessEvents(ctx context.Context, deleteChan chan watch.Ev
 
 				accessor, err := meta.Accessor(e.Object)
 				if err != nil {
-					c.logger.LogCtx(ctx, "level", "error", "message", fmt.Sprintf("cannot create accessor %T", e.Object), "stack", fmt.Sprintf("%#v", err))
+					c.logger.LogCtx(ctx, "level", "error", "message", fmt.Sprintf("failed to create accessor %T", e.Object), "stack", fmt.Sprintf("%#v", err))
 				} else {
 					ctx = setLoggerCtxValue(ctx, loggerKeyObject, accessor.GetSelfLink())
 					ctx = setLoggerCtxValue(ctx, loggerKeyVersion, accessor.GetResourceVersion())
 				}
 			}
 
+			c.logger.LogCtx(ctx, "level", "debug", "message", "reconciling object")
 			c.updateFunc(ctx, e.Object)
+			c.logger.LogCtx(ctx, "level", "debug", "message", "reconciled object")
```

Comment:
What about the methods do this logging on their own? We should then also have the event key present, which is missing now.

Reply:
> not the other way around

The resources all do it internally.